### ISSUE TITLE
Trailing slash on void element

### ISSUE THREAD

**Issue by Rotzbua** [mdn/yari#11091]
### Summary

From [Nu Html Checker](https://validator.w3.org/nu/): Trailing slash on void elements has no effect and interacts badly with unquoted attribute values.

### URL

https://developer.mozilla.org/en-US/docs/Glossary/Void_element

### Reproduction steps

1. Open html source code in browser and look html up or
2. Use online tool: https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FGlossary%2FVoid_element

### Expected behavior

Correct html:

```html
<meta charset="utf-8">
```

### Actual behavior

Formal wrong html:
```xhtml
<meta charset="utf-8"/>
```

### Device

Desktop

### Browser

Firefox

### Browser version

Stable

### Operating system

Windows

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by estelle** [comment]
This has been discussed https://github.com/orgs/mdn/discussions/242

We are not removing the trailing slashes. Doing so would require forking Prettier. The MDN Web Docs team has decided not to fork Prettier as it would introduce a big maintenance burden that is not sustainable. Concerns can be addressed to the Prettier project.

**Comment by pepelsbey** [comment]
> We are not removing the trailing slashes. Doing so would require forking Prettier.

@estelle not exactly, there’s a [Prettier plugin](https://github.com/awmottaz/prettier-plugin-void-html) that overrides this setting. So, if we ever reconsider this decision, it would be possible to implement it.

**Comment by pepelsbey** [comment]
> The discussion is about the examples and content. As far as I have checked, prettier is not applied to the HTML template for the website.

There’s a [.prettierrc](https://github.com/mdn/yari/blob/main/.prettierrc) config and a test running prettier check in [the packgage.json](https://github.com/mdn/yari/blob/main/package.json#L47). We apply the same code style across all MDN projects.

**Comment by caugner** [comment]
We discussed this internally, and came to the following conclusions:

- MDN's [Guidelines for writing HTML code examples](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Code_style_guide/HTML#general_guidelines_for_html_code_examples) explain that we use Prettier as a code formatter to keep the code style consistent.
- Because Prettier adds self-closing slashes, these are part of our de facto code style.
- We would like to remove self-closing slashes from void elements.
- However, since Prettier does not currently support this without a plugin, and it would require modifying almost all content, we will not proceed until Prettier supports this.

Therefore I'm closing this as won't fix.
